firstvoices/backend/search/indices/dictionary_entry_document.py

```diff
@@ -26,6 +26,8 @@ class DictionaryEntryDocument(Document):
     document_id = Text()
     site_id = Keyword()
     full_text_search_field = Text()
+
+    custom_order = Keyword()
```

Comment:
minor: Since this is dictionary entry specific field, can be moved a few lines down